README.md

```diff
@@ -1,3 +1,12 @@
+<p align="center"><img src="https://www.empirical-soft.com/logo.png"></p>
+
+<p align="center">
+	<a href="https://www.gnu.org/licenses/agpl-3.0.en.html"><img src="https://img.shields.io/badge/License-AGPL%20v3-blue.svg" alt="Affero General Public License (AGPL) with the Commons Clause"></a>
+	<a href="https://ci.appveyor.com/project/chrisaycock/empirical-lang"><img src="https://ci.appveyor.com/api/projects/status/t0l0fewb96t82bup?svg=true" alt="Build Status"></a>
```

Comment:
We should remove the AppVeyor badge from the "Building" section blow.

Also, let's also add [![Build Status](https://travis-ci.com/chrisaycock/empirical-lang.svg?branch=master)](https://travis-ci.com/chrisaycock/empirical-lang) for Travis CI.